contracts/tasks/load-accounts.ts

```diff
@@ -0,0 +1,42 @@
+import { Signer } from '@ethersproject/abstract-signer';
+import { Contract } from 'ethers';
+import { parseEther, parseUnits } from 'ethers/lib/utils';
+import { task } from 'hardhat/config';
+
+import { TASK_LOAD_ACCOUNTS } from './task-names';
+
+export const fundAccount = async (hre: any, tokenAddress: string, amount: string, to: string) => {
+  const tokenAbi = ['function transfer(address,uint256) returns (bool)', 'function decimals() view returns (uint8)'];
+
+  // We impersonate the binance exchange account on mainnet as a source of tokens
+  const funderAddress = '0x28C6c06298d514Db089934071355E5743bf21d60';
```

Comment:
I know this is the approach I sent on Discord, but you actually might want to take an approach similar to what I do on the `add-grant-round-manager` branch [here](https://github.com/dcgtc/dgrants/blob/29046d40cfcbbbb6916e6d001bae5913662d455e/contracts/test/utils.ts#L72-L83). It's more robust because since it will work for any token without worrying about Binance's or if Binance changes their wallet address (though you do need to manually add support per-token)

If you rebase on top of that branch you could just import those methods and use them directly

Reply:
Awesome, I've gone with this approach 🙌 - resolved in 5871fbb